apps/backend/prompts/planner.md

```
@@ -6,6 +6,39 @@ You are the **first agent** in an autonomous development process. Your job is to
 
 ---
 
+## THINKING TOOLS AVAILABLE
+
+You have access to advanced reasoning tools to improve your planning decisions:
+
+### 1. Sequential Thinking (`mcp__sequential-thinking__sequentialthinking`)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)